client/src/components/Feed/StyledCommentButton.js

```diff
@@ -0,0 +1,25 @@
+import { theme } from "constants/theme";
+import { Button } from "antd";
+import styled from "styled-components";
+
+const { darkGray, lightGray, royalBlue, white } = theme.colors;
+
+export const StyledCommentButton = styled(Button)`
+  color: ${darkGray};
+  border-color: transparent;
+  border-radius: 1rem;
+  :hover{
+    color: ${royalBlue};
+    border: 1px solid ${white};
+  }
+`;
+
+export const StyledLoadMoreButton = styled(Button)`
```

Comment:
You can extend from previous StyledCommentButton and only add the border-color rule.

Reply:
I thought about doing this but I figured it'd be best to keep them loosely coupled since they're buttons in different components and different areas of the page. It's an easy fix regardless and I can change it if you feel strongly about it. Thanks for taking a look!

Reply:
Let's go the current way.